epinorm/norm.py

```diff
@@ -266,12 +353,29 @@ def __init__(self, data_file):
         super().__init__(data_file)
 
     def _compile_location(self, record):
-        places = record["original_location"].split(":", 1)
-        location = {
-            "country": places[0].strip(),
+        """
+        extract the information from the original_location field (according to the genbank format)
+        the format is:
+        country[:region [, location]]
+
+        we will extract them into a json object with fields:
+        {
+            country
+            areas
         }
-        if len(places) > 1:
-            location["area"] = places[1].strip()
+        """
+
+        location = {"country": None, "areas":[]}
+
+        if ":" in record["original_location"]:
+            objects = record["original_location"].split(":", 1)
+
+            location["country"] = objects[0]
+            location["areas"] = objects[1].split(",") # even if there isn't a comma you get what we need
```

Comment:
I would suggest using variables names more explicit than "objects":

```suggestion
            country, area = record["original_location"].split(":", 1)
            location["country"] = country
            location["areas"] =area.split(",") # even if there isn't a comma you get what we need
```